server/src/main/java/org/opensearch/index/translog/transfer/FileSnapshot.java

```diff
@@ -108,13 +121,23 @@ public static class TransferFileSnapshot extends FileSnapshot {
 
         private final long primaryTerm;
         private Long checksum;
+        @Nullable
+        private String metadataFileName;
```

Comment:
Is this metadataFileName or checkpoint filename? In terms of remote translog, metadata has a different meaning.

Reply:
This is checkpointFileName. I was trying to make it more generic for class TransferFileSnapshot. But changing it since this might creates confusion.